tests/rgen/Microsoft.Macios.Generator.Tests/Classes/Data/ExpectedMethodsTests.cs

```diff
@@ -317,17 +321,42 @@ public virtual unsafe partial void CompleteRequest (global::Foundation.NSExtensi
 		return ret;
 	}
 
+	[SupportedOSPlatform ("macos")]
+	[SupportedOSPlatform ("ios")]
+	[SupportedOSPlatform ("tvos")]
+	[SupportedOSPlatform ("maccatalyst13.1")]
+	[BindingImpl (BindingImplOptions.GeneratedCode | BindingImplOptions.Optimizable)]
+	public static unsafe partial global::Foundation.NSData GetBookmarkData (global::Foundation.NSUrl bookmarkFileUrl, string subdomain, out global::Foundation.NSError? error)
+	{
+		if (subdomain is null)
+			global::ObjCRuntime.ThrowHelper.ThrowArgumentNullException (nameof (subdomain));
+		var bookmarkFileUrl__handle__ = bookmarkFileUrl!.GetNonNullHandle (nameof (bookmarkFileUrl));
+		var nssubdomain = global::CoreFoundation.CFString.CreateNative (subdomain);
+		global::ObjCRuntime.NativeHandle error__handle__;
```

Comment:
@rolfbjarne  this shows how we fixed the issue. The priority queue ensures that we call the GetNonNullHandle **before** we get the Handle for the nsstring. By splitting the args and their conversions, we can use the priority queue with an order and call the message with another